gatsby-node.js

```diff
@@ -253,4 +260,39 @@ exports.sourceNodes = function () {
   return function (_x3, _x4) {
     return _ref3.apply(this, arguments);
   };
-}();
\ No newline at end of file
+}();
+
+exports.createResolvers = function (_ref6) {
+  var createResolvers = _ref6.createResolvers;
+
+  var resolvers = {};
+  references.forEach(function (reference) {
+    resolvers[reference.parent] = (0, _defineProperty3.default)({}, reference.uid, {
+      resolve: function resolve(source, args, context, info) {
+        if (source[reference.uid + '___NODE']) {
+          var nodesData = [];
+          context.nodeModel.getAllNodes().find(function (node) {
+            source[reference.uid + '___NODE'].forEach(function (id) {
+              if (node.id === id) {
+                nodesData.push(node);
+              }
+            });
+          });
+          return nodesData;
+        }
+        return [];
+      }
+    });
+  });
+  groups.forEach(function (group) {
+    resolvers[group.parent] = (0, _defineProperty3.default)({}, group.field.uid, {
```

Comment:
I believe this is the root cause of the remaining problem. References and groups can have the same parent, or multiple references/groups can have the same parent. In these cases, we are overwriting the previous resolver...

Reply:
See https://github.com/contentstack/gatsby-source-contentstack/pull/43/commits/a7354b414550346a379817b29a25fc749061c29f for a potential fix. This worked in my case...

Reply:
Thanks @jbertel2 for notifying this issue and fixing it. We appreciate your contribution here.